src/components/Player/Player.jsx

```diff
@@ -38,6 +41,8 @@ const getPlayerSubroute = (info, playerId) => {
       return <ProsPage playerId={playerId} />;
     case 'rankings':
       return <RankingsPage playerId={playerId} />;
+    case 'histograms':
+      return <HistogramsPage playerId={playerId} histogramName={subInfo} />;
```

Comment:
do you still need this passed if the histogram components can read state from the route?